release/pkg/test/testdata/main-bundle-release.yaml

```diff
@@ -323,7 +323,7 @@ spec:
         imageDigest: sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
         name: eks-anywhere-cli-tools
         os: linux
-        uri: public.ecr.aws/release-container-registry/eks-anywhere-cli-tools:v0.11.0-eks-a-v0.0.0-dev-build.1
+        uri: public.ecr.aws/release-container-registry/eks-anywhere-cli-tools:v0.11.1-eks-a-v0.0.0-dev-build.1
```

Comment:
none of these changes are due to the go update. These files have to change almost every time we update the `release` folder since they depend on whatever is defined in the build tooling repo

Reply:
@abhay-krishna we should think a better long term solution here